docs/api/profiles.md

```diff
@@ -0,0 +1,86 @@
+# Profiles
+
+Profiles contain information about a User. A User does not need a Profile, but a Profile is always created
+together with a User.
+
+## Get the profile of a user
+
+### URL
+
+```http
+GET /profiles/:profile HTTP/1.1
+```
+
+### Request
+
+```http
+GET /profiles/profile_23 HTTP/1.1
+Accept: application/json
+```
+
+### Response
+
+```http
+HTTP/1.1 201 Created
+Content-Type: application/json
+
+{
+  "id": "profile_23",
+  "firstName": "Juriaan",
+  "infix": null,
+  "lastName": "Winkens",
+  "gender": "Male",
+  "birthDate":"1996-16-07T00:00:00Z",
+  "created": "2017-01-03T12:16:55Z",
+  "updated": "2017-01-03T12:16:55Z"
+}
+```
+
+## List profiles
+
+List all profiles in the active organisation.
```

Comment:
A profile should be accessed through user, not directly. I don't see a case for just listing all profiles.